src/utils/web3.ts

```diff
@@ -114,3 +101,34 @@ export function addOceanToWallet(
     }
   )
 }
+
+export function addOceanToWallet(
+  config: ConfigHelperConfig,
+  web3Provider: any
+): void {
+  const tokenMetadata = {
+    type: 'ERC20',
+    options: {
+      address: config.oceanTokenAddress,
+      symbol: config.oceanTokenSymbol,
+      decimals: 18,
+      image:
+        'https://raw.githubusercontent.com/oceanprotocol/art/main/logo/token.png'
+    }
+  }
+  addTokenToWallet(web3Provider, tokenMetadata)
+}
+
+export function addDatatokenToWallet(ddo: DDO, web3Provider: any): void {
```

Comment:
we don't need to pass around the whole DDO when only `ddo.dataTokenInfo` is needed. So could be simply something like this:

```tsx
import { DataTokenInfo } from '@oceanprotocol/lib/dist/node/ddo/interfaces/DataTokenInfo'

function addDatatokenToWallet(token: DataTokenInfo, web3Provider: any) {}
```

But then it looks like `addOceanToWallet()` & `addDatatokenToWallet()` are doing exactly the same thing, only with different values. So might be worth looking into a generic `addToken()` method accepting `address`, `symbol`, `decimals`, `image` as the parameters for it

Reply:
Yeah good point, I'll refactor that

Reply:
this has also been done